docs/get-started/getting-started-on-prem.md

```diff
@@ -668,6 +684,9 @@ non-standard location.
   The main difference between get and put is the direction of transfer. A send operation under 
   `get_zcopy` will really be `RDMA READ` from the receiver, whereas the same send will be 
   `RDMA_WRITE` from the sender if `put_zcopy` is utilized.
+- `UCX_RC_RX_QUEUE_LEN=1024` and `UCX_UD_RX_QUEUE_LEN=1024`: Length of receive queue for `rc` and 
+  `ud` transports respectively. The length change is recommended as it has shown better performance 
```

Comment:
`ud` transport is not defined, and it's not clear why they matter if the user is not requesting those transports in `UCX_TLS`.

Reply:
@yosefe, as far as I understand, `ud` is still required whenever `rc` is set, is that correct?

Reply:
yes, it is used implicitly for address exchange, when the connection establishment is to worker address. it would not be used if we use dc or if we use client/server based connection establishment.
We can use a more generic setting `UCX_IB_QUEUE_LEN=1024` to affect both and cause less confusion

Reply:
sorry, had a type: it should be `UCX_IB_RX_QUEUE_LEN` 😥